app/models/induction_period.rb

```diff
@@ -22,14 +22,20 @@ class InductionPeriod < ApplicationRecord
 
   scope :for_teacher, ->(teacher) { where(teacher:) }
   scope :for_appropriate_body, ->(appropriate_body) { where(appropriate_body:) }
-  scope :siblings_of, ->(instance) { for_teacher(instance.teacher).excluding(instance) }
+
+  # Instance methods
+  def siblings
+    return self.class.none unless teacher
```

Comment:
If it's not in a shared place is it clearer to return `InductionPeriod.none` here when there's no `teacher`?